src/admin/index.js

```diff
@@ -371,60 +371,88 @@ module.exports = ({
   /**
    * @param {string} groupId
    * @param {string} topic
+   * @param {string[]} topics
```

Comment:
I'm not sure how to write the JSDoc for this, now that the interplay between `topic` and `topics` is more complicated.

Both `topic` and `topics` are optional. If neither is set, the default is for `topics` to get the value `[]`. Do you know if there's any way to express that in JSDoc, @ankon? Not a big deal, but I just don't know.

Reply:
Interesting, from the top of my head: I think this might be too much to express with JSDoc. There could be a way to do it in the TypeScript definition, and then you could do magic with `@type` here. 

But it also might not be needed: If these are optional, then from the callers point that's all they need to know. The specific defaulting could just be considered an implementation detail.

I guess I would just go with
```js
/**
 * Function does stuff.
 *
 * Note that `topics` will be defaulted to `[]` if neither `topic` nor `topics` is provided.
 * 
 * @param {string} groupId
 * @param {string} [topic]
 * @param {string[]} [topic]
 */
```

Reply:
I added comments as @ankon suggested, inline in the parameter list as well.